tests/test_pyfftw_version.py

```diff
@@ -0,0 +1,28 @@
+import re
+
+import pyfftw
+
+
+def test_fftw_version_pattern():
+    fv = pyfftw.fftw_version
+    if fv != '':
+        match = re.search(r'(\d+).(\d+).(\d+)', fv)
+        assert match is not None
+    else:
+        # cannot get version from fftw
+        assert True
+
+
+def test_fftw_version_tuple():
+    fvt = pyfftw.fftw_version_tuple
+    if fvt:
+        # major, minor, release
+        assert len(fvt) == 3
```

Comment:
I what thinking about something like that:

```suggestion
        assert fvt > (0, 0, 0)
```

Reply:
@paugier Thanks for the suggestion, I agree the test would be more helpful by checking the content of the tuple. I think if pyfftw supports only fftw 3, would `fvt > (3, 0, 0)` be a good choice?

Thanks for pointing out that tuple can be compared in this way!